### DISCUSSION TITLE
No impressions for Sponsored Products campaign

### DISCUSSION THREAD

**Discussion by noufalos** [amzn/ads-advanced-tools-docs#127] [Q&A]
Why my campaign have no impressions?
hello guys Why i've create to many ads campaigns with different targeting types (automatique and manual) but all those campaigns doesn't get anything no impressions no click its look like they are not active even their statut is "Running"

how to fix this problem ?

**Comment by elizafarley** [comment]
Hi @noufalos, it seems like your question is around Sponsored Products campaigns. 

Did you take a look at our guides for [automatic](https://advertising.amazon.com/API/docs/en-us/guides/sponsored-products/get-started/auto-campaigns) and [manual](https://advertising.amazon.com/API/docs/en-us/guides/sponsored-products/get-started/manual-campaigns) campaign creation? 

Note that in order for your campaign to serve, you need to create a campaign, ad group, and product ad.  For manual targeting campaigns, you also need to create at least one keyword or product targeting expression.
